styles/index.css

```diff
@@ -0,0 +1,26 @@
+header {text-align: center;}
+
+ul {
+    list-style-type: none;
+}
+section.info {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    
+
```

Comment:
Delete unnecessary whitespace here before the closing bracket to improve code readability